package.json

```diff
@@ -44,6 +44,7 @@
     "ajv-error-tree": "0.0.5",
     "astring": "^1.8.4",
     "chalk": "^5.0.1",
+    "fswin": "^3.23.311",
```

Comment:
> Couple of questions/remarks:
> 
>     * Could you elaborate on when/how the short path shown its ugly head?

There are a couple of places where the code does sanity checks on code paths (for example [here](https://github.com/getappmap/appmap-agent-js/blob/e876f95881b1d8a7e66b32fca169794a585e1c2d/components/setup/node/index.mjs#L129)) and because it arrives at the path through different methods it can cause one to be short and the other to be long (see for example [this error](https://github.com/getappmap/appmap-agent-js/actions/runs/5500501136/jobs/10023447872#step:6:426)).

>     * Do we have to use this third-party package? I'm nervous at the idea of introducing a c++ depencency to solve what seems to be a minor problem.
>     * The path component should remain pure. We should either provide the information necessary to `toDirectoryPath` solve short paths in a pure manner. Or solve them at the callcite.

We have to call the system API; this is the only way to resolve a short path from a long one and vice-versa. It's not possible to do it in a pure way since the path mapping is stored in the filesystem and the only way to access it is through this API. And fswin is just a very thin wrapper on the relevant APIs. The alternative would be to write it ourselves :)